samples/snippets/export_test.py

```diff
@@ -48,10 +79,7 @@ def example_sink():
 
     yield sink
 
-    try:
-        sink.delete()
-    except Exception:
-        pass
+    _delete_sink(sink)
```

Comment:
We should handle the exception otherwise we will have flaky tests. Applies throughout

Reply:
My bad. It looks like this is already being done